src/templates/data/database/sqlserver/migration.ejs

```diff
@@ -0,0 +1,17 @@
+
+exports.up = async function (knex) {
+    knex.schema.hasTable('<%- props.table %>').then(function(exists) {
+        if(!exists) {
```

Comment:
to avoid many indentations level, we can invert the if logic like
``` js
if(exists) return
return knex.schema
           .createTable('<%- props.table %>', function (table) {
                table.bigInteger('id').primary()
                <%- props.columns %>
           })
        }

Reply:
done